constants.py

```diff
@@ -870,20 +870,25 @@
 
 kMotorPulleyGearRatio = 60 / 18 * 4 / 1
 
-kPulleyGearPitchDiameter = 1.504
-"""inches"""
+kPulleyGearPitchDiameter = 1.504 * kMetersPerInch
+"""meters"""
 
 kBottomPositionBeltPosition = 0
-kAmpPositionBeltPosition = 19.125
-kTopPositionBeltPosition = 26.5
-"""inches"""
+kAmpPositionBeltPosition = 19.125 * kMetersPerInch
+kTopPositionBeltPosition = 26.5 * kMetersPerInch
+"""meters"""
 
 kBeltPullDownSpeed = 3
 """inches per second"""
 
 kPullDownBandLimit = 0.1
 """Revolutions"""
 
+kElevatorPositionKey = "ElevatorPosition"
+
+kElevatorManualChange = 0.05
+""".05 meters per execute tick"""
```

Comment:
technically you can say this is .05m / (1/50)s so 2.5 m/s, which feels fast, I'd change these defaults